### DISCUSSION TITLE
The "model" registry in mmdet did not set import location.

### DISCUSSION THREAD

**Discussion by abeleehx** [open-mmlab/mmdetection#9548] [Q&A] [answered]
1、pip install -U openmim
2、mim install mmengine
3、mim install "mmcv>=2.0.0rc1"
4、git clone https://github.com/open-mmlab/mmdetection.git -b 3.x
5、cd mmdetection
6、pip install -v -e 
7、mim download mmdet --config yolov3_mobilenetv2_8xb24-320-300e_coco --dest .
8、python demo/image_demo.py demo/demo.jpg yolov3_mobilenetv2_8xb24-320-300e_coco.py yolov3_mobilenetv2_320_300e_coco_20210719_215349-d18dff72.pth --device cpu --out-file result.jpg

output：

12/30 17:25:05 - mmengine - WARNING - The "model" registry in mmdet did not set import location. Fallback to call `mmdet.utils.register_all_modules` instead.
12/30 17:25:05 - mmengine - WARNING - The "task util" registry in mmdet did not set import location. Fallback to call `mmdet.utils.register_all_modules` instead.
Loads checkpoint by local backend from path: yolov3_mobilenetv2_320_300e_coco_20210719_215349-d18dff72.pth
12/30 17:25:05 - mmengine - WARNING - The "visualizer" registry in mmdet did not set import location. Fallback to call `mmdet.utils.register_all_modules` instead.
/root/anaconda3/envs/openmmlab/lib/python3.8/site-packages/mmengine/visualization/visualizer.py:170: UserWarning: `Visualizer` backend is not initialized because save_dir is None.
  warnings.warn('`Visualizer` backend is not initialized '
12/30 17:25:05 - mmengine - WARNING - The "transform" registry in mmdet did not set import location. Fallback to call `mmdet.utils.register_all_modules` instead.

**Comment by RangiLyu** [comment, marked as answer]
The latest mmengine updated the registry import logic. We will support this new feature in the next version of mmdetection. You can ignore this warning currently.
